gis/GPXLab/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
+
+github.setup        BourgeoisLab GPXLab v0.7.0
+categories          gis graphics
+platforms           darwin
+license             GPL-3
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         Program to show and manipulate GPS tracks
+long_description    GPXLab is an application to display and manage GPS tracks \
+                    previously recorded with a GPS tracker.
+
+homepage            https://github.com/BourgeoisLab/GPXLab
```

Comment:
This is the `homepage` set by the GitHub portgroup already, so delete this redundant line.

Reply:
Removed.